e2e/Microsoft.Azure.Devices.E2ETests/TwinE2ETests.cs

```diff
@@ -661,5 +728,84 @@ static public void ClassCleanup()
             TestContext.WriteLine("verified " + serviceTwin.Properties.Reported[propName].ToString() + "=" + propValue);
             await TestUtil.RemoveDeviceAsync(deviceInfo.Item1, registryManager);
         }
+
+        private async Task _Twin_ServiceDoesNotCreateNullPropertyInCollection(Client.TransportType transport)
```

Comment:
Not sure why the old code has this but we shouldn't start method names with `_`.

Reply:
Fixed all